html/about.html

```diff
@@ -0,0 +1,129 @@
+      <li class="nav-items"> <a href="./index.html">Home</a></li>
+      <li class="nav-items"> <a href="./index.html">Categories</a></li>
+      <li class="nav-items"> <a href="#Events">Events</a></li>
+      <li class="nav-items"> <a href="#partner">Partner</a></li>
+    </ul>
+  </nav>
+
+
+
+  <section>
+    <div class="header">
+      <span class="subtitle">Welcome! To the Land of the Green</span>
+      <h1>Agro Food Summit 2022</h1>
+      <div id="header-text">
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vero iusto molestias harum eligendi reprehenderit
+          quidem labore modi optio aspernatur. Recusandae ducimus eos, laboriosam sint eveniet accusamus quis nisi
+          fugiat
+          dolorem.</p>
+      </div>
+      <div class="further-question">
+        <h5>Please contact us via Email for any further questions about Agro Food Summit 2022!<h5>
+            <a href="#">Agrofoodsummit@gmail.com</a>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <div class="second-header">
+      <h1>Agro Food Summit 2022 logo</h1>
+      <p>The logo of Agro Food Summit 2022 was decided through the logo competition from 10 to 12 July</p>
+
+      <div class="logo-details">
+        <img src="../images/page-logo-removebg-preview.jpg" alt="logo">
+        <h6>Creative Commons Global Summit 2010 Logo </h6>
+      </div>
+
+    </div>
+  </section>
+
+  <section class="section1">
+    <div class="container">
+      <h2>See the past Global Summit</h2>
+      <p>
+        Take a look at the last two Agricultural food Summit which took place
+        in Nigeria and Ghana.
+      </p>
+
+      <div class="history" id="Events">
+        <div class="book">
+          <img src="../images/pastsumit.jpg" class="hi-img" alt="bootcamp" />
+          <div class="info">
+            <h3>2020</h3>
+            <p class="text-white">
+              Agricultural food summit 2020 in Accra, Ghana
+            </p>
+          </div>
+        </div>
+
+        <div class="book">
+          <img src="../images/pastsummit.jpg" class="hi-img" alt="bootcamp" />
+          <div class="info">
+            <h3>2021</h3>
+            <p class="text-white">
+              Agricultural food summit 2021 in Abuja, Nigeriaia
+            </p>
+          </div>
+        </div>
+      </div>
+
+
+      <div class="partners" id="partner">
+        <h4>Partner</h4>
+        <div class="partner-container">
+          <li><a href="#"><i class="fa-brands fa-google"></i></a></i></li>
+          <li><a href="#"><i class="fa-brands fa-digg"></i></a></i></li>
+          <li><a href="#"><i class="fa-brands fa-slack"></i></a></li>
+          <li><a href="#"><i class="fa-brands fa-yahoo"></i></a></li>
+          <li><a href="#"><i class="fa-brands fa-wordpress"></i></a></i></li>
+        </div>
+      </div>
+
+      <footer>
+        <div class="footer-container">
+          <img src="../images/page-logo-removebg-preview.png" alt="logo">
```

Comment:
- [ ] Your image is broken please fix that 👇 I think you should access it from the correct directory

![image](https://user-images.githubusercontent.com/75126481/177012544-c0ae0c5b-7483-4c2b-89b2-c9edd32bcfcb.png)